### PR TITLE
Add AutoFill UI tests and fix authentication steps

### LINKED ISSUE #711: XCUITests: Add Autofill test
At least a basic test to check that the feature is working correctly after all the new changes that will be landing on master :)

### PR DESCRIPTION
**Pull request by isabelrios**
Resolves #711

This PR tries to add basic tests for the Autofill feature. At least some checks to verify that the configuration works as expected

### CODE REVIEW

**Comment by devinreams** [comment]
OK! This is looking great and ready for review! Thanks @isabelrios 🙇

File: LockboxXCUITests/LockboxXCUITests.swift [hunk 1]
```diff
@@ -6,6 +6,7 @@ import XCTest
 
 let emailTestAccountLogins = "test-b62feb2ed6@restmail.net"
 let passwordTestAccountLogins = "FRCuQaPm"
+let testingURL = "https://wopr.norad.org/~sarentz/fxios/testpages/password.html"
```
Comment (joeyg) [inline LockboxXCUITests/LockboxXCUITests.swift]: I don't think this is used anywhere

File: LockboxXCUITests/LockboxXCUITests.swift [hunk 2]
```diff
@@ -258,4 +249,50 @@ class LockboxXCUITests: BaseTestCase {
...
+        waitforExistence(safari.buttons["URL"], timeout: 5)
+        waitforExistence(safari.textFields["Email or phone"])
+        safari.textFields["Email or phone"].tap()
+        // Need to confirm what is shown here, different elements have appeared and
+        if (safari.otherElements["Password Auto-fill"].exists) {
+            safari.otherElements["Password Auto-fill"].tap()
+        }
+        safari.buttons["Use “iosmztest@gmail.com”"].tap()
+        // Once previous is clear we can assert that the element shown is correct
+        //XCTAssertTrue(safari.buttons["iosmztest@gmail.com, for this website — Lockbox"].exists)
```
Comment (joeyg) [inline LockboxXCUITests/LockboxXCUITests.swift]: Why is this commented out?

File: LockboxXCUITests/LockboxXCUITests.swift (hunk 1 above)
Comment (isabelrios) [reply LockboxXCUITests/LockboxXCUITests.swift]: Yes, you are right! forgot to remove

File: LockboxXCUITests/LockboxXCUITests.swift (hunk 2 above)
Comment (isabelrios) [reply LockboxXCUITests/LockboxXCUITests.swift]: During testing I has seeing to different screen when autofill was suggested:

lockboxautofill1
lockboxautofill2


So I commented the assert until it was clear which one was shown and it was always being shown so there were not false failures.
I will check the app and try to uncomment that line if that is clear now.

File: LockboxXCUITests/LockboxXCUITests.swift (hunk 1 above)
Comment (devinreams) [reply LockboxXCUITests/LockboxXCUITests.swift]: @isabelrios Did you want `testingURL` still or shall we yank that before we merge this? I'm ready whenever you're ready. 👍

**Pull request merged**